docs/quickstart-and-usage.md

```diff
@@ -114,12 +114,15 @@ Inspect the `uds-bundle.yaml` of a bundle
 
 #### Viewing Images in a Bundle
 
-It is possible to derive images from a `uds-bundle.yaml`. This can be useful for situations where you need to know what images will be bundled before you actually create the bundle. This is accomplished with the `--list-images` flag. For example:
+It is possible to derive images from a `uds-bundle.yaml`, local UDS tarball artifacts, and remote OCI repos. This can be useful for situations where you need to know what images will be bundled before you actually create the bundle or what images will be deployed if using an already created bundle. This is accomplished with the `--list-images` flag. For example:
 
-`uds inspect ./uds-bundle.yaml --list-images`
+`uds inspect --list-images [BUNDLE_YAML_FILE|BUNDLE_TARBALL|OCI_REF]`
 
 This command will return a list of images derived from the bundle's packages, taking into account optional and required package components.
 
+> !Warning *Breaking Change*  
```

Comment:
IMO this should go in the release notes (or at least in the Slack message that we send out to announce the latest release). I would add an example of the output though; and if you want to be really nice add the `yq` command for parsing just the images